PhysicsTools/PythonAnalysis/test/BuildFile.xml

```diff
@@ -0,0 +1,6 @@
+<environment>
```

Comment:
@davidlange6 , there is no need to have <environment> </environment> tag. 
If you want to run these tests whenever we update matplotlib or histogrammar then please add dependency on py2-matplotlib and histogrammar.